public/apps/configuration/app-router.tsx

```diff
@@ -147,7 +147,7 @@ export function AppRouter(props: AppDependencies) {
               )}
             />
             <Route
-              path={buildUrl(ResourceType.roles, Action.view, ':roleName')}
+              path={buildUrl(ResourceType.roles, Action.view, ':roleName', ':prevAction?')}
```

Comment:
Wondering in this case `/roles/view/roleName` still works or not.

Reply:
Yes, it works.
![image](https://user-images.githubusercontent.com/63824432/90451796-7bd2f200-e0a1-11ea-9b38-f7b059fb0f64.png)